EIPS/eip-1.mediawiki

```diff
@@ -86,45 +83,42 @@ Each EIP should have the following parts:
 
 * Implementations -- The implementations must be completed before any EIP is given status "Final", but it need not be completed before the EIP is accepted. It is better to finish the specification and rationale first and reach consensus on it before writing code.
 
+* Simple Summary -- "If you can't explain it simply, you don't understand it well enough." Provide a simplified and layman-accessible explanation of the EIP.
+
 ==EIP Formats and Templates==
 
-EIPs should be written in mediawiki or markdown format. Image files should be included in a subdirectory for that EIP.
+EIPs should be written in [https://en.wikipedia.org/wiki/Help:Cheatsheet mediawiki] or [https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet markdown] format. Image files should be included in a subdirectory for that EIP.
```

Comment:
That's nothing compared to MediaWiki. _Parsing_ MW markup is, believe it or not, turing complete, because you cannot parse it without the set of plugins that it was written for. In fact, mediawiki doesn't really have a parser - it has a list of regular expression search-and-replace operations.